samples/automl/automlTranslationDataset.js

```diff
@@ -335,26 +324,61 @@ require(`yargs`)
       description: `The target language to be translated to`,
     },
   })
-  .command(`createDataset`, `creates a new Dataset`, {}, opts =>
-    createDataset(
-      opts.projectId,
-      opts.computeRegion,
-      opts.datasetName,
-      opts.source,
-      opts.target
-    )
+  .command(
+    `createDataset`,
+    `creates a new Dataset`,
+    {},
+    async opts =>
+      await createDataset(
+        opts.projectId,
+        opts.computeRegion,
+        opts.datasetName,
+        opts.source,
+        opts.target
+      ).catch(console.error)

```

Comment:
Interesting.  What happens if you don't make any of these changes in the `yargs` section?

Reply:
In case of error, functions will return promise reject.
i think its good to handle error with catch block in this section, we can remove async and await if needed.